manifest.json

```
@@ -1,12 +1,10 @@
 {
   "vendor": "vtex",
   "name": "order-placed",
-  "version": "2.18.0",
+  "version": "2.18.1",
```

```suggestion
  "version": "2.18.0",
```

O CI/CD faz a atualização de forma automática depois de escolher o versionamento